package.json

```diff
@@ -84,7 +84,7 @@
     "copy-webpack-plugin": "5.0.3",
     "cross-env": "5.2.0",
     "css-loader": "2.1.1",
-    "enzyme": "3.9.0",
+    "enzyme": "3.10.0",
```

Comment:
Upgrading enzyme because of https://github.com/airbnb/enzyme/issues/2020 which was needed for adding couple of test cases and also fix https://mattermost.atlassian.net/browse/MM-15837.